fe/src/components/DividerLine.vue

```diff
@@ -0,0 +1,12 @@
+<template>
```

Comment:
I think we can use a global class instead of making a component for just a stand alone divider here so please make a css file for that.

Reply:
resolved it maam